### PR TITLE
Barcode DPI 

### LINKED ISSUE #9293: [Solution found, please implement]1D barcode resolution (DPI) too low when Label dimensions are low (Not scannable)
#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode 
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
Hi,
we have a 9 symbol long asset number "SYS000000" and when we print lables 1.6inch (42mm) long, the barcodes are not scannable. After days of debugging I noticed that when I "generate lables" with a Lengh of 4inch, then set the Label dimensions back to 1.6 inch and printed them, it suddenly got scannable (because previously generated high-resolution barcodes are cached even when small dimention is printed).
Later on I've seen that the generated public/uploads/barcodes/c128-sysXXXXXX.png file has a resolution relative to the Label dimensions.
If I by mistake generate the lables first with 1.6 inch, I need to delete them in public/uploads/barcodes/. Then re-generate at 4 inch -> set back to 1.6 inch -> 're-generate' the lables to print them with the cached high-res barcode

edit: Forgot to mention that this issue appeared with all 1d barcode types
My custom css to fit barcode qr code and text together (Vertically)
[Custom CSS (Branding).txt](https://github.com/snipe/snipe-it/files/6203192/Custom.CSS.Branding.txt)
image
**Expected behavior**
Is it possible to set the barcode generation resolution relative to the symbols encoded and not to the lable lengh?
Or just a very high hardcoded resolution so this won't happen, even when the lables are 'smal'?

**Screenshots**
generated at 1.6 inch setting (not enough resolution/DPI to get distinct distinct bar sizes)
barcode (1876) at 1 6inch
Generated at 4 inch setting (when printed also in 1.6 inch, it's scannable as DPI is high enough for correct bar sizes)
barcode (1876) at 4inch

2021-03-15 14_42_20-Update Label Settings __ Papyrus Asset management

**Server (please complete the following information):**
 - Snipe-IT Version: v5.0.11 build 5695
 - OS: Ubuntu 18.04
 - Web Server: Apache
 - PHP Version 7.2.34-2+ubuntu18.04.1
 - IMAGE_LIB= tested both gd & imagick

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome, FireFox - Irrelevant

**Error Messages**
- no Error as it's only noticible when scanning barcode 

**Additional context**
- also tested with fresh install (manual and install .sh)

btw: Thanks for the amazing SnipeIT project <3
-Alex

### PR DESCRIPTION
**Pull request by AlexanderWPapyrus**
# Description
Hi this is the correct commit instead #9343

Motivation-> Issue #9293
CSS used to fit the barcode, qr code and Text tighter together
[Custom CSS (Branding).txt](https://github.com/snipe/snipe-it/files/6203126/Custom.CSS.Branding.txt)
image
image

Fixes #9293

Type of change:
Adjusted multiplier so barcode is readable even with low "label width"
snipeit\app\Http\Controllers\Assets\AssetsController.php
changed 96.000000000001 to 200.000000000001
`                // Calculate barcode width in pixel based on label width (inch)` 
`                 $barcode_width = ($settings->labels_width - $settings->labels_display_sgutter) * 200.000000000001;` 

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Printed lables out with 1.6 inch width and they are scannable as the generated Barcode PNG has enough resolution for the saved barcode data.

Lables look the same so no formatation issue with the additional width of the barcode, just better DPI.

This is not a real fix but an optimization as the dpi should be relative to the encoded data and not to the width of the lable, but it allows to use smaller lables without any deficits (minimaly more diskspace per barcode image but if you'd print 2" lables it would be anyways that big)

**Test Configuration**: ^above
* PHP version: 7.4.16
* Webserver version Apache
* OS version: ubuntu 18.04.5 LTS
* SnipeIT: v5.1.4 build 5886 (g9f3a8a43c)

# Checklist:
Basically 'just' changed a multiplyer so no logical change
- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [ ] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by snipe** [comment]
Hi there - thanks for this! Can you please re-target this PR to point to the `develop` branch, per [our Contributing documentation](https://snipe-it.readme.io/docs/contributing-overview)? 

You don't need to close and re-open. After you create a pull request, you can modify the base branch so that the changes in the pull request are compared against a different branch. By changing the base branch of your original pull request rather than opening a new one with the correct base branch, you’ll be able to keep valuable work and discussion.

mhuxq

Thanks!

**Comment by snipe** [comment]
Also please rebase - there are some LDAP changes in this PR that make it very difficult to see what changed

**Comment by AlexanderWPapyrus** [comment]
Hi,
thanks, changed base to dev, now there's only my changed
app/Http/Controllers/Assets/AssetsController.php

**Comment by snipe** [comment]
Thanks!!
